Added support badge [#168917413]

dougmartin opened the pull request.

kswenson commented on src/lib/db-listeners/db-supports-listener.ts:
```diff
@@ -34,6 +39,10 @@ export class DBSupportsListener extends BaseListener {
     }
+    if (this.lastSupportViewTimestampRef) {
+      this.debugLogHandler("#stop", "removing", "on value", this.lastSupportViewTimestampRef);
+      this.lastSupportViewTimestampRef.on("value", this.handleLastSupportViewTimestampRef);
```
```suggestion
      this.lastSupportViewTimestampRef.off("value", this.handleLastSupportViewTimestampRef);
```
Shouldn't we be `off`ing in stop()?

dougmartin replied:
Yes, that is a copy/paste error :( - good catch.

kswenson commented on src/clue/clue.sass:
```diff
@@ -37,6 +37,17 @@
+    left: -$padding - ($badge-radius / 4)
+    top: 0
+    background-color: #fff
+    border: 4px solid $sky
```
I'm not seeing any handling of the `:hover` state here. In the prototype it switches to blue with a white outline on hover.

lbondaryk replied:
perhaps, although I'm not all that concerned about that. Only that it should disappear once the tab is opened.

dougmartin commented:
@kswenson I fixed the ref `off` copy/paste error and added the hover styling to the badge.  I had to add hover state to the tab as just adding `:hover` to the support wouldn't change it when the parent tab itself was hovered over.  The hover state looks like this now:  

![image](https://user-images.githubusercontent.com/112938/66836799-00a82900-ef30-11e9-8384-049f7f313019.png)

NOTE: we are still missing the shadow on the badge.  We would need to change the badge to a .png with the shadow since adding a `box-shadow` to the element would put the shadow over the tab edge.
